src/components/booksComponent/AddBookForm.js

```diff
@@ -1,14 +1,65 @@
-import React from 'react';
-
-const AddBookForm = () => (
-  <div>
-    <h2>Add New Book</h2>
-    <form className="form">
-      <input type="text" placeholder="Book Title" />
-      <input type="text" placeholder="Categorie" />
-      <button type="submit">Add Book</button>
-    </form>
-  </div>
-);
+import { React, useState } from 'react';
+import PropTypes from 'prop-types';
+
+// import { useDispatch } from 'react-redux';
+// import { addBook } from '../../redux/books/books';
```

Comment:
- [ ] Please delete any part of code that's not in use and is commented out, they are several in this particular file. It makes the codebase dirty.